ruby_event_store/spec/mappers/default_spec.rb

```diff
@@ -0,0 +1,46 @@
+SomethingHappened = Class.new(RubyEventStore::Event)
+
+module RubyEventStore
+  module Mappers
+    RSpec.describe Default do
+      let(:data)   {{some_attribute: 5}}
+      let(:metadata)     {{some_meta: 1}}
+      let(:domain_event) { SomethingHappened.new(data: data, metadata: metadata) }
+
+      specify '#event_to_serialized_record returns YAML serialized record' do
+        record = subject.event_to_serialized_record(domain_event)
+        expect(record).to            be_a SerializedRecord
+        expect(record.id).to         eq domain_event.event_id
+        expect(record.metadata).to   eq YAML.dump(metadata)
+        expect(record.data).to       eq YAML.dump(data)
+        expect(record.event_type).to eq SomethingHappened
```

Comment:
You can see an incompatible interface and I asked for `String`. Here it is a `Class` but in next test it is a `String`. It should be the same thing always.